website/docs/language/resources/ephemeral/index.mdx

```diff
@@ -44,21 +44,43 @@ Use write-only arguments to securely pass temporary values to resources during a
 <CodeBlockConfig highlight="11">
 
 ```hcl
-ephemeral "random" "password" {
+ephemeral "random_password" "db_password" {
   length = 16
 }
 
-resource "aws_db_instance" "test" {
-  instance_class      = "db.t5.micro"
+resource "aws_secretsmanager_secret" "db_password" {
+  name = "db_password"
+}
+
+resource "aws_secretsmanager_secret_version" "db_password" {
+  secret_id                = aws_secretsmanager_secret.db_password.id
+  secret_string_wo         = ephemeral.random_password.db_password.result
+  secret_string_wo_version = 1
+}
+
+ephemeral "aws_secretsmanager_secret_version" "db_password" {
+  secret_id = aws_secretsmanager_secret.db_password.id
+}
+
+resource "aws_db_instance" "example" {
+  instance_class      = "db.t3.micro"
   allocated_storage   = "5"
   engine              = "postgres"
-  username            = "admin"
+  username            = "example"
   skip_final_snapshot = true
-  password_wo         = ephemeral.random.password.value
+
+  password_wo         = ephemeral.aws_secretsmanager_secret_version.db_password.secret_string
   password_wo_version = 1
 }
 ```
 
+### Deferring ephemeral resources
```

Comment:
Since this is explaining the code block above, I think we should remove the header to keep the context of that code block in mind:

```suggestion
```